mapfishapp/README.md

```diff
@@ -56,10 +56,21 @@ It is also possible to POST a JSON string to the home controller, for instance :
             "layername": "ign:ign_bdtopo_region",
             "owstype": "WMS",
             "owsurl": "http://ids.pigma.org/geoserver/ign/wms"
-        }]
+        }],
+        "search":{
+    		"owsurl":"http://ids.pigma.org/geoserver/ows",
+    		"cql_filter": "id_dept = 48",
+    		"featurens":"http://ids.pigma.org/geoserver/ign",
+    		"typename":"ign_bdtopo_departement"
```

Comment:
I'm not sure you need to ask for `featurens` here if the external app provides you with the fully qualified layer name (`nsalias:typename`).
That would also be more in line with the other "layer" inputs above.